react/src/components/BAITag.tsx

```diff
@@ -0,0 +1,29 @@
+import { ConfigProvider, Tag } from 'antd';
+import { TagProps } from 'antd/lib/tag';
+import React from 'react';
+
+interface BAITagProps extends TagProps {}
+
+const BAITag: React.FC<BAITagProps> = ({ ...tagProps }) => {
+  return (
+    <ConfigProvider
+      theme={{
+        components: {
+          Tag: {
+            borderRadiusSM: 11,
+            colorText: '#999999',
+            defaultBg: 'transparent',
+            colorInfoBg: 'transparent',
+            colorWarningBg: 'transparent',
+            colorErrorBg: 'transparent',
+            colorSuccessBg: 'transparent',
+          },
+        },
+      }}
+    >
+      <Tag {...tagProps} />
+    </ConfigProvider>
+  );
+};
+
+export default BAITag;
```

Comment:
Set the padding same as [SessionStatusTag](https://github.com/lablup/backend.ai-webui/blob/88f1c2bed2cc0164891ded0068e7e33159419af5/react/src/components/ComputeSessionNodeItems/SessionStatusTag.tsx#L79-L82).